azurelinuxagent/pa/rdma/ubuntu.py

```diff
@@ -79,7 +79,7 @@ def install_driver(self):  # pylint: disable=R0912,R0911
         status,output = shellutil.run_get_output('apt-cache show --no-all-versions linux-azure')
         if status != 0:
             return
-        r = re.search('Version: (\S+)', output)  # pylint: disable=W1401,C0103
+        r = re.search('Version: (\S+)', output)  # pylint: disable=W1401
```

Comment:
Ah, this `W1401=anomalous-backslash-in-string` is I think a missed warning suppression; we had a PR removing this warning from the comment block in `ci/*.pylintrc`. With the move to `pylint --list-msgs` this is actually not a big deal, so I don't think we need to take any action (i.e. creating a ticket to fix it) besides our usual linter hunting.

Thoughts?